app/views/pages/sponsors.html.erb

```diff
@@ -67,3 +67,4 @@
     </div>
   </div>
 </div>
+<%= javascript_packs_with_chunks_tag "followButtons", defer: true %>
```

Comment:
Same question as above about a container element and `data-follow-button-container="true"`